webpack.config.dev.js

```diff
@@ -6,7 +6,11 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const path = require('path');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 
-const noVisualization = process.env.ANALYSE_PACK.toString() === 'false';
+if (process.env.ANALYSE_PACK) {
```

Comment:
change the logic. Now you made a const inside the if. The variable will be destroyed as sum as the if ends